PocketKit/Sources/PocketKit/MyList/SearchItemsList/LocalSearch.swift

```diff
@@ -0,0 +1,27 @@
+// This Source Code Form is subject to the terms of the Mozilla Public
+// License, v. 2.0. If a copy of the MPL was not distributed with this
+// file, You can obtain one at http://mozilla.org/MPL/2.0/.
+
+import Sync
+
+class LocalSearch {
```

Comment:
```suggestion
class LocalSavesSearch {
```

Just to clarify that it's a local search _for saves_, and not archived items.

Reply:
good point, updated! I'll request her review as well - thanks for the quick response!